docs/guided-tour/target-maps/03-multiple-subinst/README.md

```diff
@@ -0,0 +1,100 @@
+    target clusters.
+
+The templates for these resources can be found [here](component/installation) and can be deployed with 
+this [script](component/commands/deploy-k8s-resources.sh). Adapt the [settings](component/commands/settings) file
+such that the entry `TARGET_CLUSTER_KUBECONFIG_PATH` points to the kubeconfig of the target cluster to which the
+config maps should be deployed.
+
+Now you should see successful installations on your Landscaper resource cluster:
+
+```
+kubectl get inst -n cu-example 
+NAME                               PHASE       EXECUTION                          AGE
+multiple-subinst                   Succeeded                                      17s
+multiple-subinst-sub-blue-hhdmz    Succeeded   multiple-subinst-sub-blue-hhdmz    16s
+multiple-subinst-sub-green-55nw2   Succeeded   multiple-subinst-sub-green-55nw2   16s
+multiple-subinst-sub-red-srm8g     Succeeded   multiple-subinst-sub-red-srm8g     16s
+```
+
+Let's have a deeper look into the resources of the example. The root Installation `multiple-subinst` is quite similar
+to the first and second target map example and contains as import a target map importing three of the five deployed 
+targets as well as configuration data.
+
+The root installation references this [blueprint](component/blueprint/root/blueprint.yaml) which creates a Subinstallation 
+for every target. This is done [here](component/blueprint/root/subinst-execution.yaml), where the following expression 
+loops over all input targets:
+
+```
+{{ range $key, $target := .imports.rootclusters }}
+```
+
+Every Subinstallation gets as import one target from the target map with the following expression:
+
+```
+    imports:
+      targets:
+        - name: cluster
+          target: rootclusters[{{ $key }}]
+```
+
+In the import data mapping part, the key is forwarded to the [blueprint](component/blueprint/sub/blueprint.yaml) of 
+the Subinstallations such that the DeployItems can get a stable name, which is important for a later deletion if
+particular targets are removed. Furthermore, the correct entry from the data is extracted and provided:
+
+```
+    importDataMappings:
+      instanceName: {{ $key }}
+      config:
+        {{- $config | toYaml | nindent 8 }}
+```
+
+Every Subinstallation creates one DeployItem to deploy one config map.
+
+Now you can see successful installations on your Landscaper resource cluster:
+
+```
+kubectl get di -n cu-example                                                  
+NAME                                    TYPE                                            PHASE       EXPORTREF   AGE
+multiple-subinst                   Succeeded                                      17s
+multiple-subinst-sub-blue-hhdmz    Succeeded   multiple-subinst-sub-blue-hhdmz    16s
+multiple-subinst-sub-green-55nw2   Succeeded   multiple-subinst-sub-green-55nw2   16s
+multiple-subinst-sub-red-srm8g     Succeeded   multiple-subinst-sub-red-srm8g     16s
+```
```

Comment:
There is something wrong here. The text says "installations", the command says "di", and the result table is the same as above.

Reply:
I have fixed this such that now the DIs are presented here.